tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "rootDir": "./src",
     "baseUrl": "./",
     "outDir": "./dist",
-    "lib": ["es2017"]
+    "lib": ["es2017", "dom"]
```

Comment:
why do we need dom ?! :open_mouth:

Reply:
I thought you would say this! Apparently there's an issue with superagent https://github.com/DefinitelyTyped/DefinitelyTyped/issues/12044 that requires the dom library. I didn't like it but it was the quickest fix but yea. I probably shouldn't have added it. Another method is to create a file declaring the `Blob` and `XmlHttpRequest` interfaces and I didn't like that neither

Reply:
Ok, That sounds weird but ok.

Reply:
I don't think the dom library is a good idea tbh

Reply:
me too, but anyway we need to investigate more to see what cusses this issue.
is that a typescript issue ?